db/migrate/20171128040545_create_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTransactions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :transactions do |t|
+      t.integer :invoice_id
+      t.string :credit_card_number
```

Comment:
good idea, I actually saved the unit prices as integers and I think I might end up changing it in a second